### PR TITLE
#3133 Add a timeline message when the IEC WT is disconnected because …

### PR DESCRIPTION
**Pull request by louisg-rte**
…of its protection

**Checklist before requesting a review**

*use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes*

- [ ] unit tests and non-regression tests were added (new model, new feature and bug fix)
- [ ] main documentation was updated (update of input/output file, 3rd party, model, repository organization, solver)
- [ ] example documentations were updated (new example in examples folder)
- [ ] the corresponding milestone was added in the ticket and in this PR
- [ ] if this PR modifies the parameters or inputs/outputs of a model/solver: the corresponding xsl was added in util/xsl and platform DB were updated
- [x] if this PR modifies a dictionary: the corresponding french dictionary was updated

### CODE REVIEW

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/IEC/BaseControls/Auxiliaries/GridProtection.mo [hunk 1]
```diff
@@ -85,6 +88,18 @@ model GridProtection "Grid protection system for wind turbines (IEC N°61400-27-
     Dialog(tab = "Operating point"));
 
 equation
+  when greater.y then
```
Comment (rosiereflo) [inline dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/IEC/BaseControls/Auxiliaries/GridProtection.mo:91]: when greater.y and not pre(greater.y)?

**Review by rosiereflo — CHANGES_REQUESTED** [review]

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/IEC/BaseControls/Auxiliaries/GridProtection.mo (hunk 1 above)
Comment (adriguir) [reply dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/IEC/BaseControls/Auxiliaries/GridProtection.mo:91]: As it is a when, doesn't it natively encompass such a condition, ie the idea that you have a transition?
I don't remember if there are cases in which we need to have the pre()

Comment (rosiereflo) [reply dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/IEC/BaseControls/Auxiliaries/GridProtection.mo:91]: I'm not sure anymore :) do we have a testcase in which one of those values become true?

Comment (louisg-rte) [reply dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/IEC/BaseControls/Auxiliaries/GridProtection.mo:91]: [TEST2.zip](https://github.com/dynawo/dynawo/files/15010232/TEST2.zip)
I'm using this test case to test this feature

Comment (rosiereflo) [reply dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/IEC/BaseControls/Auxiliaries/GridProtection.mo:91]: Je vais faire 2-3 tests pour voir si le pre est necessaire. En attendant est ce qu'on pourrait ajouter ce cas test aux nrt de dynawo-rte ? merci :)

Comment (louisg-rte) [reply dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/IEC/BaseControls/Auxiliaries/GridProtection.mo:91]: Should I also include the obtained timeline in the reference outputs ?

Comment (rosiereflo) [reply dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/IEC/BaseControls/Auxiliaries/GridProtection.mo:91]: yes please :)

**Review by rosiereflo — APPROVED** [review]

**Pull request merged**